### PR TITLE
Bundle update

### PR DESCRIPTION
**Pull request by k2nr**
Just updated gems with `bundle update`. Some tests failed due to [this change](https://github.com/thoughtbot/factory_bot/blob/master/NEWS#L3) so I fixed them

### CODE REVIEW

**Comment by k2nr** [comment]
@essa can you review?

File: spec/factories/districts.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@
     sequence :name do |n|
       "district#{n}"
     end
+    region { 'us-east-1' }
```
Comment (essa) [inline spec/factories/districts.rb]: Why is this needed?

Comment (k2nr) [reply spec/factories/districts.rb]: As the factory_bot's changelog explains, the default strategy has been changed so that associations are by default "built" instead of "create". Previously associations were created and [this method https://github.com/degica/barcelona/blob/master/app/models/district.rb#L177] is called before validation. But now the method is not called since associations are just built.

The other spec change is made for the same reason.

Comment (essa) [reply spec/factories/districts.rb]: I see, thanks!

**Comment by essa** [comment]
LGTM 👍

**Comment by k2nr** [comment]
Thanks!
